tests/verifyDashboards_test.js

```diff
@@ -0,0 +1,26 @@
+Feature('Test Dashboards');
```

Comment:
Please rename Feature and file itself to something more explicit. Like `PT Summary` or something similar